Client/Clique/Modules/EventDetail/EventDetailViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  EventDetailViewController.swift
+//  Clique
+//
+//  Created by Infinum on 18.11.2022..
+//
+
+import Foundation
+import UIKit
+
+class EventDetailViewController: UIViewController{
+    var eventGet : Event?
+    
+    @IBOutlet private var _eventDescription: UILabel!
+    @IBOutlet private var _eventName: UILabel!
+    @IBOutlet private var _eventImage: UIImageView!
+    @IBOutlet private var _eventCreator: UILabel!
+    @IBOutlet private var _eventLocation: UILabel!
+    @IBOutlet private var _eventTimestamp: UILabel!
+    @IBOutlet private var _eventParticipantsNum: UILabel!
+    @IBOutlet private var _eventCost: UILabel!
+    @IBOutlet private var _eventCategory: UILabel!
+    
+    override func viewDidLoad() {
+        
+        super.viewDidLoad()
+        _setupUI()
+        
+    }
```

Comment:
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
        _setupUI()
    }
```

You do not need so much spacing around methods here.